jdl/jhipster/default-application-options.js

```diff
@@ -227,6 +234,14 @@ function getDefaultConfigForNewApplication(customOptions = {}) {
   if (!options[PACKAGE_FOLDER] && options[PACKAGE_NAME]) {
     options[PACKAGE_FOLDER] = options[PACKAGE_NAME].replace(/\./g, '/');
   }
+  let skipClient = options[SKIP_CLIENT];
+  if (skipClient === undefined) {
+    skipClient = options[CLIENT_FRAMEWORK] === undefined || options[CLIENT_FRAMEWORK] === NO;
+  }
+  if (skipClient) {
+    delete options[CLIENT_FRAMEWORK];
```

Comment:
Should not remove clientFramework.
Please set clientFramework instead (related to https://github.com/jhipster/generator-jhipster/blob/main/rfcs/5-jhipster-rfc-inclusive-property-in-generator.md)
```suggestion
    options[CLIENT_FRAMEWORK] = NO;
```
Related to https://github.com/jhipster/generator-jhipster/blob/main/rfcs/5-jhipster-rfc-inclusive-property-in-generator.md